bitcoinology/what-is-lightning-network.md

```diff
@@ -0,0 +1,90 @@
+
+## The Bolts
+
+So the bolts are the standards. They're a bit like RFCs for those of you who are engineers or internet people. They're a bit like RFCs. Basically describing what the protocol should do, what the protocol must do, what every software must agree to in order for all these lightning off-chain payments to work. And the outcome of all that is, in 2017, the software moves forward, they have something that kind of works, and we get by the summer of 2017, this sort of historic moment, it's kind of historic, kind of silly in a way, which is at Room 77 in Berlin you have Laulu, one of the engineers I just mentioned, paying for a beer with Lightning to, this is Jorg Platzer, the guy who owns, owned Room 77, actually with Testnet Lightning. So he didn't actually use real Bitcoin. He used testnet There's actually kind of a story behind that because why did they use? Why did they use testnet? I wonder if I can ask the audience Why did they use testnet lightning in the middle of 2017 and not mainchain Bitcoin lightning? Because Segwit hadn't been adopted at that point. Because Segwit hadn't been adopted at that point, thank you, that's the correct answer. So the weird thing is on Litecoin, they actually activated this Segwit thing, this thing that fixes malleability, some months earlier. So they were even trying to do this stuff on Litecoin, as well as on, this is confusing, Litecoin Lightning. They were doing it on Litecoin and on Bitcoin testnet, but they weren't doing it on main chain Bitcoin, because 2017 was like the end of the block size war, it was this massive explosion. I can't even begin to describe 2017 to you, It was the most crazy year ever from a whole bunch of different angles. But from one angle was we finally activated SegWit at the end of August, somebody correct me, or the beginning of August 2017, middle of August, Michael, anyone? Roughly, whatever, August 2017. So we activated SegWit in the midst of the hundred other things happening, which I'm not going to describe. But it meant that from the end of 2017, we could finally use this off-chain protocol called Lightning on the Bitcoin mainnet. Now, It didn't mean that everyone thought it was a great idea, but a lot of people did and it went forward. 
```

Comment:
```suggestion
So the bolts are the standards. They're a bit like RFCs for those of you who are engineers or internet people. They're a bit like RFCs. Basically describing what the protocol should do, what the protocol must do, what every software must agree to in order for all these lightning off-chain payments to work. And the outcome of all that is, in 2017, the software moves forward, they have something that kind of works, and we get by the summer of 2017, this sort of historic moment, it's kind of historic, kind of silly in a way, which is at Room 77 in Berlin you have Laolu, one of the engineers I just mentioned, paying for a beer with Lightning to, this is Jorg Platzer, the guy who owns, owned Room 77, actually with Testnet Lightning. So he didn't actually use real Bitcoin. He used testnet There's actually kind of a story behind that because why did they use? Why did they use testnet? I wonder if I can ask the audience Why did they use testnet lightning in the middle of 2017 and not mainchain Bitcoin lightning? Because Segwit hadn't been adopted at that point. Because Segwit hadn't been adopted at that point, thank you, that's the correct answer. So the weird thing is on Litecoin, they actually activated this Segwit thing, this thing that fixes malleability, some months earlier. So they were even trying to do this stuff on Litecoin, as well as on, this is confusing, Litecoin Lightning. They were doing it on Litecoin and on Bitcoin testnet, but they weren't doing it on main chain Bitcoin, because 2017 was like the end of the block size war, it was this massive explosion. I can't even begin to describe 2017 to you, It was the most crazy year ever from a whole bunch of different angles. But from one angle was we finally activated SegWit at the end of August, somebody correct me, or the beginning of August 2017, middle of August, Michael, anyone? Roughly, whatever, August 2017. So we activated SegWit in the midst of the hundred other things happening, which I'm not going to describe. But it meant that from the end of 2017, we could finally use this off-chain protocol called Lightning on the Bitcoin mainnet. Now, It didn't mean that everyone thought it was a great idea, but a lot of people did and it went forward. 
```